collect_app/src/main/res/layout/file_manager_layout.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 Nafundi
```

Comment:
In this case, @shobhitagarwal1612, you are actually the copyright holder. It depends on the context of the work if it's done for hire, etc. Can you please put your name there?

The goal in the future is to transfer the rights to the "ODK organization" and not need to have these copyright headers on every file but we're not there quite yet!